.github/actions/npm/action.yml

```diff
@@ -5,12 +5,6 @@ runs:
   steps:
     - run: npm i
       shell: bash
-      if: contains('
-        refs/heads/master
-        refs/heads/release-please--branches--master
-        ', github.ref)
+      if: true
     - uses: bahmutov/npm-install@v1
```

Comment:
@weboko I believe this github action also provide some caching. Disabling means possible slow down of the CI